structures/CommandStore.js

```diff
@@ -0,0 +1,41 @@
+const { Collection } = require("discord.js");
+
+class CommandStore extends Collection {
```

Comment:
This name is misleading its not a "Store" its a Collection

Reply:
What about `UserStore` extends `DataStore` extends `Collection` extends `Map`, which is a thing? We use this pattern in discord.js and several frameworks. -1 in that, @Dev-Yukine.

Reply:
well because all are extending each other, other than this. The Class is called Command**Store** but does not extend DataStore rather than extending collection so when someone who knows d.js reads the name "CommandStore" he/she could be confused and think this class instance did extend DataStore and use methods like .resolve() (what this class do not have) on it and it wouldn't have any effect